### ISSUE TITLE
Identity Center Application Assignments needed

### ISSUE THREAD

**Issue by tparvu** [aws/aws-sdk-go-v2#2319]
### Describe the feature

A resource that allows one to assign groups and users to an Identity Center Application is needed. 

### Use Case

I am forced to use click-ops.

I am required to use IaC.

Therefore, I can not use AWS Identity Center.

### Proposed Solution

Add a resource that can assign users and groups to a SSO Application.

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### AWS Go SDK V2 Module Versions Used

Release (2023-10-17)

### Go version used

go version go1.21.3 linux/amd64

**Comment by lucix-aws** [comment]
This seems related but not necessarily identical to https://github.com/aws/aws-sdk-go-v2/issues/2294.

TL;DR we don't have control over support for this. AWS SDKs are generated from a set of models owned by service teams.

Please consider filing a support ticket through the AWS console to request this feature.
